labs/optimization/optimization_plots.py

```diff
@@ -19,7 +19,7 @@ def plot_contour(f, allvecs, legend_path):
     plt.rc("text", usetex=False)
     plt.rc("font", family="serif")
     plt.plot(1, 1, "r*", markersize=10, label="minimum")
-    plt.plot(4.5, -1.5, "bx", markersize=10, label="initial guess")
+    plt.plot(initial_guess[0], initial_guess[1], "bx", markersize=10, label="initial guess")
```

Comment:
Try automatic unpacking here? https://www.geeksforgeeks.org/packing-and-unpacking-arguments-in-python *inital_guess should do the trick